### PR TITLE
[BACKLOG-6818] - Text rotation feature

### PR DESCRIPTION
**Pull request by EgorZhuk**

### CODE REVIEW

File: engine/core/source/org/pentaho/reporting/engine/classic/core/style/TextRotation.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import org.pentaho.reporting.engine.classic.core.util.ObjectStreamResolveException;
+
+import java.io.ObjectStreamException;
+import java.io.Serializable;
+
+public class TextRotation implements Serializable {
+  public static final TextRotation D_90 = new TextRotation( "90" );
+  public static final TextRotation D_270 = new TextRotation( "-90" );
+  private static final String CSS =
+    "transform: rotate(#deg); -ms-transform: rotate(#deg); -webkit-transform: rotate(#deg);";
```
Comment (tmorgner) [inline engine/core/source/org/pentaho/reporting/engine/classic/core/style/TextRotation.java]: The use of # as magic constant might store trouble for future maintainer. Replace # with {0} and use "MessageFormat.format(CSS, String.valueOf(getNumericValue()));" and you never have to explain the meaning of # in this context to any other developer.

**Comment by EgorZhuk** [comment]
More checkstyle.

**Comment by EgorZhuk** [comment]
@tmorgner is it OK, or more tests are needed?

**Comment by tmorgner** [comment]
The coverage changes are all over the place, so I attribute it to the normal randomness of whatever happens inside the other tests. Coverage is good enough for me. Lets merge ...

**Pull request merged**